<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimConnectSettingsComponent</class>
 <widget class="QWidget" name="CSimConnectSettingsComponent">
  <property name="windowTitle">
   <string>FSX/P3D SimConnect config</string>
  </property>
  <layout class="QVBoxLayout" name="vl_SettingsComponent">
   <item>
    <widget class="QGroupBox" name="gb_FsxSimConnectConfigFile">
     <property name="title">
      <string>FSX/P3D config files</string>
     </property>
     <layout class="QFormLayout" name="fl_Settings">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_SimConnect">
        <property name="text">
         <string>SimConnect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="pte_SimConnectInfo">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="documentTitle">
         <string>Version info</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cb_P3DVersion">
        <property name="currentText">
         <string>6.0 64bit</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>6.0 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.2 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.1 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.0 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.5 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.4 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.3 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.2 64bit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.1 64bit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_P3DVersion">
        <property name="text">
         <string>P3D 64 bit version</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_P3DVersion</tabstop>
  <tabstop>pte_SimConnectInfo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
